### ISSUE TITLE
Images broken/missing in preview on Safari

### ISSUE THREAD

**Issue by andyw8** [scrimba/community#630]
Safari Version 15.2 (17612.3.6.1.6)
Apple M1 Pro

In the Learn React course, the images appear as a broken icon in the browser preview, e.g. at the –2:25 timestamp in https://scrimba.com/learn/learnreact/project-setup-navbar-cob544349a4494c348ab844f0

<img width="421" alt="Screen Shot 2021-12-29 at 12 46 34 PM" src="https://user-images.githubusercontent.com/13400/147689631-1daf6dba-d509-4786-b36f-175e3dfde7f7.png">

I see the following warning in Safari's console, not sure if related.

> Blocked script execution in 'https://scrimba.com/__sw__blank.html' because the document's frame is sandboxed and the 'allow-scripts' permission is not set.

It works fine in Chrome.

**Comment by eighteentee** [comment]
Yes! I thought I was going mad. Has there been any movement on this? Using Chrome or Edge to run these Scrimba courses chews through battery - Safari sips battery.

**Comment by aolite** [comment]
Totally agree! I noticesd same bug in Safari browser... Are any solution to that?

Thanks in advance!

**Comment by eighteentee** [comment]
Using anything but Safari when away from a power source (even on my M1 Max) results in reduced battery life. This is a shame that Safari isn't supported. Maybe the Scrimba admins should put a disclaimer on the site to say that Scrimba isn't compatible with Safari if it's not going to be fixed?

**Comment by snbns** [comment]
Has anyone figured out how to resolve this issue? I'm seeing the same thing while doing Scrimba courses.

**Comment by Brandonagil** [comment]
still exist

**Comment by whoisyulu** [comment]
I want to use Scrimba on iPad Pro but non of the major browsers works (including Chrome). 

All browsers on iOS/iPad OS are Webkit based and given Safari on Mac is not working as well, I'd guess this is has something todo with Webkit.
